index.html

```diff
@@ -80,16 +80,28 @@ <h2>Add New Recipe</h2>
 
     <div id="image-container"></div>
 
-    <form id="search" type="submit">
+    <form id="search" type="add recipe">
```

Comment:
```suggestion
    <form id="search">
```
See previous comment about form's type.